stages/07.extra-tweaks/02.install-rpi-packages/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# kuiper2.0 - Embedded Linux for Analog Devices Products
+#
+# Copyright (c) 2024 Analog Devices, Inc.
+# Author: Larisa Radu <larisa.radu@analog.com>
+
+if [ "${INSTALL_RPI_PACKAGES}" = y ]; then
+	
+	if [ "${CONFIG_DESKTOP}" = y ]; then
+		# Copy application launcher and icon for sense_emu
+		install -d 								"${BUILD_DIR}/usr/local/share/sense"
```

Comment:
Yes, sense_emu_gui.desktop ensures that sense_emu is seen as an application. It can be started from terminal by running "sense_emu" or by graphical interface. The application can be found under Applications/Development folder.